### PR TITLE
Feature/optional history

### PR DESCRIPTION
**Pull request by joswarmer**
Add option to use a database without any history, the current tests run about 20% to 25% slower with history compared to without history. Note that the history tabel is still there, and the repo version is till part of t he response. This way clients can see whether there have been changes since some previous time. 
Having no history means that only the current nodes are stored in the database, so the history API will not work.

Technically this is done by creating different tables at database initialisation time.

NOTE: the first eight commits are because of a rebase upon development.

### CODE REVIEW

File: packages/test/src/test/tests.ts [hunk 1]
```diff
@@ -9,417 +9,419 @@ import sm from "source-map-support"
...
-            console.log("Cannot store initial chunk: " + JSON.stringify(result.body))
-        }
-        // console.log("STORE PARTITIONS RESULT " + JSON.stringify(result.body.messages))
-        const tmp2 = result.body.messages.find(m => m.data["version"] !== undefined).data["version"]
-        console.log("TMP 2 [" + tmp2 + "]")
-        baseFullChunkVersion = Number.parseInt(result.body.messages.find(m => m.data["version"] !== undefined).data["version"])
-        console.log("repoVersionAfterPartitionCreated " + initialPartitionVersion + "repoVersionAfterPartitionFilled " + baseFullChunkVersion)
-    })
+// Run all, tests with and without history
+collection.forEach(withoutHistory =>
```
Comment (dslmeinte) [inline packages/test/src/test/tests.ts:15]: Nice! (`collection` could be inlined.)

**Review by dslmeinte — APPROVED** [review]
(but see remarks)

File: packages/test/src/test/tests.ts (hunk 1 above)
Comment (joswarmer) [reply packages/test/src/test/tests.ts:15]: Strangely enough I get a TypeScript error message when I inline the collection: TS2538: Type false cannot be used as an index type

File: packages/dbadmin/src/tools/init-tables-no-history-sql.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import {
+    CONTAINMENTS_TABLE,
+    NODES_TABLE,
+    PROPERTIES_TABLE,
+    REFERENCES_TABLE,
+    RESERVED_IDS_TABLE
+} from "@lionweb/repository-common";
+
+export const INIT_TABLES_SQL_WITHOUT_HISTORY = `
```
[the remark this answers is not in the document]
Comment (joswarmer) [reply packages/dbadmin/src/tools/init-tables-no-history-sql.ts:9]: It is a large SQL string, and just enough differences to make this hard. Therefore I left it as two separate strings.

File: packages/test/src/test/tests.ts (hunk 1 above)
Comment (dslmeinte) [reply packages/test/src/test/tests.ts:15]: Hmmm, weird...but also not really important :)

File: packages/dbadmin/src/tools/init-tables-no-history-sql.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (dslmeinte) [reply packages/dbadmin/src/tools/init-tables-no-history-sql.ts:9]: Yeah, it certainly wouldn't be trivial. Let's see how these files evolve and find out whether it's worthwhile after all to do the DRYing.

**Pull request merged**